### DISCUSSION TITLE
[Support] 依赖包reflect-metadata 版本升级导致构建报错

### DISCUSSION THREAD

**Discussion by microTT** [midwayjs/midway#3498] [Q&A] [answered]
### Describe the problem（描述问题）

reflect-metadata 12月13日发布了 0.2.0 版本

如果工程下其他包依赖了版本较高的新版本
![image](https://github.com/midwayjs/midway/assets/6782488/77de671e-4b10-421d-b6d6-47beba282575)

并且其他包优先 import "reflect-metadata"
会导致 mideway 使用高版本的 0.2.0版本的 reflect-metadata

构建的时候会报错
![image](https://github.com/midwayjs/midway/assets/6782488/c305a854-51dd-4abf-9547-c99dcb91317f)

### Midway Versions（Midway 版本）

>> Start to check your midway component version...

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(3.12.5)
✓ @midwayjs/async-hooks-context-manager(3.13.5)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(not installed)
✓ @midwayjs/bull(not installed)
✓ @midwayjs/bull-board(not installed)
✓ @midwayjs/cache(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.13.5)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(not installed)
✓ @midwayjs/decorator(3.13.5)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(3.13.5)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(not installed)
✓ @midwayjs/info(not installed)
✓ @midwayjs/jwt(not installed)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.13.5)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(not installed)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(3.13.5)
✓ @midwayjs/session(not installed)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(not installed)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(not installed)
✓ @midwayjs/upload(not installed)
✓ @midwayjs/validate(not installed)
✓ @midwayjs/version(3.13.5)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(not installed)
✓ @midwayjs/ws(not installed)
**********************************************************************
>> Check complete, all versions are healthy.
**********************************************************************

**Comment by czy88840616** [comment]
已监控到 reflect-metadata 的更新，正在评估中。

**Reply by czy88840616** [reply, marked as answer]
已发布 v3.13.7 ，将 reflect-metadata 升级到 0.2.1
